### DISCUSSION TITLE
allow locking for indefinite time but enforce delay when attempting to unlock

### DISCUSSION THREAD

**Discussion by distbit0** [10x-development/lockmeout.online#4] [Ideas]
Hi it would be very useful to be able to lock a password for an indefinite time but such that if I would like to access it I have to wait e.g. one day to do so, during which I can chose to cancel the unlock process. many thx!

**Comment by Dynalon** [comment]
Hi, thanks for your feedback. Agreed, I had a similar idea to add it soon. Would it be important to configure the time for you - i.e. such that you can set a time window in hours for how long to wait (1h, 4h, 24h,48h and so on)?

**Comment by distbit0** [comment]
Yes that would be very useful. also important that the unlock delay period can be cancelled once it has been started, in case I change my mind. many thx

**Comment by Dynalon** [comment]
This is implemented as of today. You can select "Infinite" when creating a lock and chose a countdown duration. The countdown durations for now are static, let me know if that works for you, or if you prefere other countdown intervals.
